stable/polaris/templates/audit.job.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.audit.enable -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    checksum/config: '{{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}'
+  name: {{ include "polaris.fullname" . }}
+  {{- if .Values.templateOnly }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    {{- include "polaris.labels" . | nindent 4 }}
+    component: audit
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ include "polaris.fullname" . }}-audit
+      restartPolicy: Never
+      containers:
+      - command:
+        - polaris
+        - --audit
+        - --output-url
+        - {{ .Values.audit.outputURL }}
```

Comment:
Can you add some info to the values file or to the readme describing what expected `outputURL` values look like?

Reply:
Agreed - this breaks installing by just setting `audit.enable=true` with the following error: 

```
UPGRADE FAILED
Error: error validating "": error validating data: ValidationError(Job.spec.template.spec.containers[0].command): unknown object type "nil" in Job.spec.template.spec.containers[0].command[3]
Error: UPGRADE FAILED: error validating "": error validating data: ValidationError(Job.spec.template.spec.containers[0].command): unknown object type "nil" in Job.spec.template.spec.containers[0].command[3]
```

Reply:
It may be a better error to add a `required` function if enabled is true. Example

```gotemplate
        - --output-url
        - {{ require "Must set audit.outputURL in values if you enable the audit job." .Values.audit.outputURL }}
        - --output-file
```

Reply:
Thanks Nick! I didn't know about the `required` function - works much better now.